### DISCUSSION TITLE
this is trash

### DISCUSSION THREAD

**Discussion by Cbo11** [Babyhamsta/Aimmy#1320] [General]
you should tell these people to fuck off and not make cheats for games and to play the game without breaking terms of service and w/o cheating im deleting aimmy because it sucks and it breaks terms of service thats why it wont work in fortnite + who even plays fortnite. so just stop making cheats and play the game with respect for the devs

**Comment by substanceabuse666** [comment]
bro is mad because he cant run hax , git gud retardo

**Comment by natdog400** [comment]
> you should tell these people to fuck off and not make cheats for games and to play the game without breaking terms of service and w/o cheating im deleting aimmy because it sucks and it breaks terms of service thats why it wont work in fortnite + who even plays fortnite. so just stop making cheats and play the game with respect for the devs

https://github.com/SunOner/yolov8_aimbot use this its way better and will be open source forever you closet cheater lol

**Comment by ghost** [comment]
🤓🤓

**Comment by that1guy20** [comment]
> you should tell these people to fuck off and not make cheats for games and to play the game without breaking terms of service and w/o cheating im deleting aimmy because it sucks and it breaks terms of service thats why it wont work in fortnite + who even plays fortnite. so just stop making cheats and play the game with respect for the devs

Wait, I'm confused 😕 are you saying you didn't know that you were getting a cheat program before you decided to click download and install? Nor did you read about it? you know, the part that says it can get you banned.  What a Douchbag  just go to bed
